targetFunctions/+domainsTF/customXY.m

```diff
@@ -13,11 +13,19 @@
     % Extract individual parameters from problemStruct
     [numberOfContrasts, ~, contrastBackgroundIndices, contrastQzshiftIndices,...
      contrastScalefactorIndices, contrastBulkInIndices, contrastBulkOutIndices,...
-     contrastResolutionIndices, contrastDomainRatioIndices, backgroundParamArray,...
+     contrastResolutionParamIndices, contrastDomainRatioIndices, backgroundParamArray,...
      qzshiftArray, scalefactorArray, bulkInArray, bulkOutArray, resolutionParamArray,...
      domainRatioArray, dataPresent, nParams, params, ~, ~, contrastBackgroundActions,...
      cCustFiles, useImaginary] = extractProblemParams(problemStruct);
 
+    numDomainRatios = length(domainRatioArray);
```

Comment:
Is this missed or can it not be moved?

Reply:
Good job someone is paying attention . . .